samples/python/digits_video.py

```diff
@@ -41,7 +45,7 @@ def main():
 
         bin = cv.adaptiveThreshold(gray, 255, cv.ADAPTIVE_THRESH_MEAN_C, cv.THRESH_BINARY_INV, 31, 10)
         bin = cv.medianBlur(bin, 3)
-        _, contours, heirs = cv.findContours( bin.copy(), cv.RETR_CCOMP, cv.CHAIN_APPROX_SIMPLE)
+        contours, heirs = cv.findContours( bin.copy(), cv.RETR_CCOMP, cv.CHAIN_APPROX_SIMPLE)
```

Comment:
OpenCV 3.4.x still uses 3 outputs (due compatibility reasons).

Please apply this fix on master branch (if applicable).

Reply:
ah, right, will do.

Reply:
I mean separate patch. Keep this for 3.4.

BTW, patch for master is not needed ([code](https://github.com/opencv/opencv/blob/4.2.0/samples/python/digits_video.py#L44)).